charts/app-config/values-integration.yaml

```
@@ -1007,6 +1007,21 @@ govukApplications:
 
 - name: frontend
   helmValues:
+    ingress:
+      enabled: true
+      annotations:
+        <<: *default-alb-ingress-annotations
+        <<: *alb-ingress-www-waf-ruleset
+        alb.ingress.kubernetes.io/load-balancer-name: assets-origin
+        alb.ingress.kubernetes.io/group.name: assets-origin
+        alb.ingress.kubernetes.io/group.order: '10'
```

The `whitehall-frontend` ingress already has priority 10, so we need a different number here in order to define an unambiguous ordering of rules for the load balancer. How do we choose this number?

- Besides our new `frontend` ingress and `whitehall-frontend`, there are two other ingresses in the same group (`group.name: assets-origin`) i.e. sharing the same load balancer. These are `asset-manager` and `static`.
- The path that we want to handle here at `frontend` (`/government/uploads/system/uploads/attachment_data/file/*/*/preview`) is a strict subset of one of `asset-manager`'s rules (`/government/uploads/`).
- `asset-manager` has priority 20.
- We therefore need to have a higher priority (lower number) than asset-manager.

```suggestion
        alb.ingress.kubernetes.io/group.order: '15'
```

In case you're curious and want all the gory details: https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/guide/ingress/annotations/

---

Thanks.  I'll set this to 15 now.

`whitehall-frontend` will be going away within months, so we can remove that (and re-adjust the orders) at that point.